templates/production.ts

```diff
@@ -0,0 +1,368 @@
+export const PRODUCTION_ACCESS_CONSUMER_TEMPLATE = `
+<head>
+  <style>
+    .card {
+      margin: 0 0 20px 0;
+      background: #FFFFFF;
+      Padding: 24px 24px 0.1px 24px;
+      font-family: Helvetica, Arial, sans-serif;
+      font-size: 16px;
+      line-height: 25px;
+      max-width: 700px;
+    }
+
+    .header {
+      background: #112E51;
+      width: 100%;
+      height: 80px;
+    }
+
+    .header img {
+      margin-top: 30px;
+      margin-left: 20px;
+    }
+
+    .steps {
+      width: 24px;
+      float:left;
+      height: 26px;
+      margin-right: 10px;
+    }
+
+    .steps-container {
+      height: 26px;
+      margin-bottom: 20px;
+    }
+
+    .sub {
+      margin-top: -25px;
+      margin-left: 34px;
+      float:center;
+    }
+
+    h1 {
+      margin-bottom: 20px;
+      padding: 0;
+      font-size: 24px;
+      line-height: 30px;
+      font-weight: bold;
+      color: #323A45;
+    }
+
+    h2 {
+      margin-bottom: 15px;
+      padding: 0;
+      font-size: 16px;
+      line-height: 25px;
+      font-weight: bold;
+      color: #323A45;
+      display: inline;
+    }
+
+    p {
+      margin-bottom: 20px;
+      font-size: 16px;
+      line-height: 25px;
+      color: #323A45;
+    }
+  </style>
+</head>
+<div class="card">
+  <div class="header"><img src="https://dvp-developer-portal-backend-images.s3-us-gov-west-1.amazonaws.com/VA-Lighhouse-APIs-email.png" alt="logo"/></div>
```

Comment:
maybe make the alt "VA Lighthouse APIs" or "VA Lighthouse APIs logo"?

Reply:
will be in next commit